lib/schema.js

```diff
@@ -166,6 +169,64 @@ export default class Schema {
     }
   }
 
+
+  /**
+   * Creates an index on the specified property.
+   *
+   * @public
+   * @param {String} prop property to create the index on
+   * @param {Object} options index options
+   * @param {String} options.indexName
+   * @param {String} options.indexType
+   * @param {String} options.refKeyCase
```

Comment:
we're going to update docs separately. I will publish this as `next` version to npm so we can continue to work with it. Will update docs in a bit later.